tools/azure-sdk-tools/ci_tools/build.py

```diff
@@ -180,17 +180,9 @@ def create_package(
     setup_parsed = ParsedSetup.from_path(setup_directory_or_file)
 
     if setup_parsed.ext_modules:
-        run(
-            [sys.executable, "-m", "cibuildwheel", "--output-dir", dist], cwd=setup_parsed.folder
-        )
+        run([sys.executable, "-m", "cibuildwheel", "--output-dir", dist], cwd=setup_parsed.folder, check=True)
 
     if enable_wheel:
-        run_logged(
-            [sys.executable, "setup.py", "bdist_wheel", "-d", dist], prefix="create_wheel", cwd=setup_parsed.folder
```

Comment:
Was removing the prefix here and for `create_sdist` intentional?

Reply:
100$ `prefix` doesn't apply to `run()`.

Reply:
LGTM then! Thanks for the clarification